<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
  "http://www.w3.org/TR/html4/loose.dtd">

<head>
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<title>CIT Course References</title>
<style type="text/css" media="all">
  h1 {
    text-align: center;
  }
  li {
    margin-top: 1em;
  }
</style> 
</head>

<h1>CIT Course References</h1>  
<ol>
<li>Assersohn, L., Gangi, L., Zhao, Y., Dowsett, M., Simon, R., Powles, T. J., &amp; Liu, E. T. (2002). The feasibility of using fine needle aspiration from primary breast cancers for cDNA microarray analyses <i>Clinical Cancer Research</i>, <i>8</i>(3), 794-801.</li>
<li>Baldi, P., &amp; Long, A. D. (2001). A Bayesian framework for the analysis of microarray expression data: regularized t -test and statistical inferences of gene changes. <i>Bioinformatics</i>, <i>17</i>(6), 509-519.</li>
<li>Benjamini, Y., &amp; Hochberg, Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>Journal of the Royal Statistical Society. Series B</i>, <i>57</i>(1), 289-300.</li>
<li>Bittner, M., Meltzer, P., Chen, Y., Jiang, Y., Seftor, E., Hendrix, M., Radmacher, M., et al. (2000). Molecular classification of cutaneous malignant melanoma by gene expression profiling <i>Nature</i>, <i>406</i>(6795), 536-540. doi:10.1038/35020115</li>
<li>Black, M. A., &amp; Doerge, R. W. (2002). Calculation of the minimum number of replicate spots required for detection of significant gene expression fold change in microarray experiments <i>Bioinformatics</i>, <i>18</i>(12), 1609-1616.</li>
<li>Bolstad, B. M., Irizarry, R A, Astrand, M., &amp; Speed, T P. (2003). A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. <i>Bioinformatics</i>, <i>19</i>(2), 185-193.</li>
<li>Breiman, L. (2001). Random Forests. <i>Machine Learning</i>, <i>45</i>, 5-32.</li>
<li>Chen, Y., Dougherty, E. R., &amp; Bittner, M. L. (1997). Ratio-based decisions and the quantitative analysis of cDNA microarray images. <i>Journal of Biomedical Optics</i>, <i>2</i>(4), 364-374.</li>
<li>Dettling, M., &amp; Buhlmann, P. (2003). Boosting for tumor classification with gene expression data. <i>Bioinformatics</i>, <i>19</i>(9), 1061-1069. doi:10.1093/bioinformatics/btf867</li>
<li>Dobbin, K. K., &amp; Simon, R. M. (2007). Sample size planning for developing classifiers using high-dimensional DNA microarray data <i>Biostatistics</i>, <i>8</i>(1), 101-117. doi:10.1093/biostatistics/kxj036</li>
<li>Dobbin, K. K., Zhao, Y., &amp; Simon, R. M. (2008). How large a training set is needed to develop a classifier for microarray data <i>Clinical Cancer Research</i>, <i>14</i>(1), 108-114. doi:10.1158/1078-0432.CCR-07-0443</li>
<li>Dobbin, K., &amp; Simon, R. (2002). Comparison of microarray designs for class comparison and class discovery <i>Bioinformatics</i>, <i>18</i>(11), 1438-1445.</li>
<li>Dobbin, K., &amp; Simon, R. (2005). Sample size determination in microarray experiments for class comparison and prognostic classification <i>Biostatistics</i>, <i>6</i>(1), 27-38. doi:10.1093/biostatistics/kxh015</li>
<li>Dobbin, K., Shih, J H, &amp; Simon, R. (2003). Statistical design of reverse dye microarrays <i>Bioinformatics</i>, <i>19</i>(7), 803-810.</li>
<li>Dudoit, S., &amp; Fridlyand, J. (2002). A prediction-based resampling method for estimating the number of clusters in a dataset <i>Genome Biology</i>, <i>3</i>(7), RESEARCH0036.</li>
<li>Dudoit, S., Fridlyand, J., &amp; Speed, Terence P. (2002). Comparison of discrimination methods for the classification of tumors using gene expression data. <i>Journal of the American Statistical Association</i>, <i>97</i>(457), 77-87.</li>
<li>Dudoit, S., Shaffer, J. P., &amp; Boldrick, J. C. (2003). Multiple hypothesis testing in microarray experiments. <i>Statistical Science</i>, <i>18</i>(1), 71-103.</li>
<li>Efron, B., Tibshirani, Robert, Storey, J. D., &amp; Tusher, V. (2001). Empirical Bayes analysis of a microarray experiment. <i>Journal of the American Statistical Association</i>, <i>96</i>, 1151-1160.</li>
<li>Freund, Y., &amp; Schapire, R. (1999). A short introduction to boosting. <i>Journal of Japanese Society for Artificial Intelligence</i>, <i>14</i>(5), 771-780.</li>
<li>Friedman, JH. (2002). Stochastic gradient boosting. <i>Computational Statistics &amp; Data Analysis</i>, <i>38</i>, 367-378.</li>
<li>Friedman, Jerome, Hastie, T., &amp; Tibshirani, Robert. (2000). Additive logistic regression: a statistical view of boosting. <i>The Annals of Statistics</i>, <i>28</i>(2), 337-407. doi:10.1214/aos/1016218223</li>
<li>Furey, T. S., Cristianini, N., Duffy, N., Bednarski, D. W., Schummer, M., &amp; Haussler, D. (2000). Support vector machine classification and validation of cancer tissue samples using microarray expression data <i>Bioinformatics</i>, <i>16</i>(10), 906-914.</li>
<li>Geman, D., d'Avignon, C., Naiman, D. Q., &amp; Winslow, R. L. (2004). Classifying gene expression profiles from pairwise mRNA comparisons <i>Statistical Applications in Genetics and Molecular Biology</i>, <i>3</i>, Article19. doi:10.2202/1544-6115.1071</li>
<li>Geschwind, D. H. (2001). Sharing gene expression data: an array of options <i>Nature Reviews Neuroscience</i>, <i>2</i>(6), 435-438. doi:10.1038/35077576</li>
<li>Golub, T. R., Slonim, D. K., Tamayo, P., Huard, C., Gaasenbeek, M., Mesirov, J. P., Coller, H., et al. (1999). Molecular classification of cancer: class discovery and class prediction by gene expression monitoring <i>Science</i>, <i>286</i>(5439), 531-537.</li>
<li>Hedenfalk, I., Duggan, D., Chen, Y., Radmacher, M., Bittner, M., Simon, R., Meltzer, P., et al. (2001). Gene-expression profiles in hereditary breast cancer <i>The New England Journal of Medicine</i>, <i>344</i>(8), 539-548. doi:10.1056/NEJM200102223440801</li>
<li>Irizarry, R A. (2003). Summaries of Affymetrix GeneChip probe level data. <i>Nucleic Acids Research</i>, <i>31</i>(4), e15. doi:10.1093/nar/gng015</li>
<li>Irizarry, Rafael A, Hobbs, B., Collin, F., Beazer-Barclay, Y. D., Antonellis, K. J., Scherf, U., &amp; Speed, T. P. (2003). Exploration, normalization, and summaries of high density oligonucleotide array probe level data. <i>Biostatistics</i>, <i>4</i>(2), 249-264.</li>
<li>Jiang, W., Varma, S., &amp; Simon, R. (2008). Calculating confidence intervals for prediction error in microarray classification using resampling <i>Statistical Applications in Genetics and Molecular Biology</i>, <i>7</i>(1), Article8. doi:10.2202/1544-6115.1322</li>
<li>Kendziorski, C. M., Zhang, Y., Lan, H., &amp; Attie, A. D. (2003). The efficiency of pooling mRNA in microarray experiments <i>Biostatistics</i>, <i>4</i>(3), 465-477. doi:10.1093/biostatistics/4.3.465</li>
<li>Kerr, M. K., &amp; Churchill, G. A. (2001a). Bootstrapping cluster analysis: assessing the reliability of conclusions from microarray experiments <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>98</i>(16), 8961-8965. doi:10.1073/pnas.161273698</li>
<li>Kerr, M. K., &amp; Churchill, G. A. (2001b). Experimental design for gene expression microarrays <i>Biostatistics</i>, <i>2</i>(2), 183-201. doi:10.1093/biostatistics/2.2.183</li>
<li>Khan, J., Wei, J. S., Ringn√©r, M., Saal, L. H., Ladanyi, M., Westermann, F., Berthold, F., et al. (2001). Classification and diagnostic prediction of cancers using gene expression profiling and artificial neural networks <i>Nature Medicine</i>, <i>7</i>(6), 673-679. doi:10.1038/89044</li>
<li>Korn, E., Troendle, J., McShane, L., &amp; Simon, R. (2004). Controlling the number of false discoveries: application to high-dimensional genomic data. <i>Journal of Statistical Planning and Inference</i>, <i>124</i>(2), 379-398.</li>
<li>Lee, M. L., Kuo, F. C., Whitmore, G. A., &amp; Sklar, J. (2000). Importance of replication in microarray gene expression studies: statistical methods and evidence from repetitive cDNA hybridizations <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>97</i>(18), 9834-9839.</li>
<li>Li, C., &amp; Wong, Wing Hung. (2001a). Model-based analysis of oligonucleotide arrays: model validation, design issues and standard error application. <i>Genome Biology</i>, <i>2</i>(8), research0032.</li>
<li>Li, C., &amp; Wong, Wing Hung. (2001b). Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>98</i>(1), 31-36.</li>
<li>Lockhart, D. J., Dong, H., Byrne, M. C., Follettie, M. T., Gallo, M. V., Chee, M. S., Mittmann, M., et al. (1996). Expression monitoring by hybridization to high-density oligonucleotide arrays <i>Nature Biotechnology</i>, <i>14</i>(13), 1675-1680. doi:10.1038/nbt1296-1675</li>
<li>Lusa, L., McShane, L. M., Radmacher, M. D., Shih, J. H., Wright, G. W., &amp; Simon, R. (2007). Appropriateness of some resampling-based inference procedures for assessing performance of prognostic classifiers derived from microarray data <i>Statistics in Medicine</i>, <i>26</i>(5), 1102-1113. doi:10.1002/sim.2598</li>
<li>Manduchi, E., Grant, G. R., McKenzie, S. E., Overton, G. C., Surrey, S., &amp; Stoeckert, C. J. (2000). Generation of patterns from gene expression data by assigning confidence to differentially expressed genes <i>Bioinformatics</i>, <i>16</i>(8), 685-698.</li>
<li>McShane, L. M., Radmacher, M. D., Freidlin, B., Yu, R., Li, M., &amp; Simon, R. (2002). Methods for assessing reproducibility of clustering patterns observed in analyses of microarray data <i>Bioinformatics</i>, <i>18</i>(11), 1462-1469.</li>
<li>McShane, L. M., Shih, Joanna H, &amp; Michalowska, A. M. (2003). Statistical issues in the design and analysis of gene expression microarray studies of animal models <i>Journal of Mammary Gland Biology and Neoplasia</i>, <i>8</i>(3), 359-374.</li>
<li>Milligan, G. W., &amp; Cooper, M. C. (1985). An examination of procedures for determining the number of clusters in a data set. <i>Psychometrika</i>, <i>50</i>(2), 159-179. doi:10.1007/BF02294245</li>
<li>Molinaro, A. M., Simon, R., &amp; Pfeiffer, R. M. (2005). Prediction error estimation: a comparison of resampling methods <i>Bioinformatics</i>, <i>21</i>(15), 3301-3307. doi:10.1093/bioinformatics/bti499</li>
<li>Newton, M. A., Kendziorski, C. M., Richmond, C. S., Blattner, F. R., &amp; Tsui, K. W. (2001). On differential variability of expression ratios: improving statistical inference about gene expression changes from microarray data. <i>Journal of Computational Biology</i>, <i>8</i>(1), 37-52. doi:10.1089/106652701300099074</li>
<li>Radmacher, M. D., McShane, L. M., &amp; Simon, R. (2002). A paradigm for class prediction using gene expression profiles <i>Journal of Computational Biology</i>, <i>9</i>(3), 505-511. doi:10.1089/106652702760138592</li>
<li>Ritchie, M. E., Carvalho, B. S., Hetrick, K. N., Tavare, S., &amp; Irizarry, R A. (2009). R/Bioconductor software for Illumina's Infinium whole-genome genotyping BeadChips. <i>Bioinformatics</i>, <i>25</i>(19), 2621-2623. doi:10.1093/bioinformatics/btp470</li>
<li>Schadt, E. C., Li, C., Su, C., &amp; Wong, Wing H. (2000). Analyzing high‚Äêdensity oligonucleotide gene expression array data. <i>Journal of Cellular Biochemistry</i>, <i>80</i>, 192-202.</li>
<li>Schena, M., Shalon, D., Davis, R. W., &amp; Brown, P. O. (1995). Quantitative monitoring of gene expression patterns with a complementary DNA microarray <i>Science</i>, <i>270</i>(5235), 467-470.</li>
<li>Shih, Joanna H, Michalowska, A. M., Dobbin, K., Ye, Y., Qiu, T. H., &amp; Green, J. E. (2004). Effects of pooling mRNA in microarray class comparisons <i>Bioinformatics</i>, <i>20</i>(18), 3318-3325. doi:10.1093/bioinformatics/bth391</li>
<li>Simon, R. M., McShane, L. M., Wright, G. W., Korn, E. L., Radmacher, M. D., &amp; Zhao, Y. (2008). <i>Design and Analysis of DNA Microarray Investigations</i> (pp. 1-205). New York: Springer-Verlag.</li>
<li>Simon, R., Radmacher, M. D., Dobbin, K., &amp; McShane, L. M. (2003). Pitfalls in the use of DNA microarray data for diagnostic and prognostic classification <i>Journal of the National Cancer Institute</i>, <i>95</i>(1), 14-18.</li>
<li>Tibshirani, Robert, Hastie, T., Narasimhan, B., &amp; Chu, Gilbert. (2002). Diagnosis of multiple cancer types by shrunken centroids of gene expression <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>99</i>(10), 6567-6572. doi:10.1073/pnas.082099299</li>
<li>Tukey, J. W. (1993). Tightening the clinical trial <i>Controlled Clinical Trials</i>, <i>14</i>(4), 266-285.</li>
<li>Tusher, V. G., Tibshirani, R, &amp; Chu, G. (2001). Significance analysis of microarrays applied to the ionizing radiation response. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, <i>98</i>(9), 5116-5121. doi:10.1073/pnas.091062498</li>
<li>Wright, G. W., &amp; Simon, R. M. (2003). A random variance model for detection of differential gene expression in small microarray experiments. <i>Bioinformatics</i>, <i>19</i>(18), 2448-2455.</li>
<li>Wu, Z., Irizarry, Rafael A, Gentleman, R., Martinez-Murillo, F., &amp; Spencer, F. (2004). A Model-Based Background Adjustment for Oligonucleotide Expression Arrays. <i>Journal of the American Statistical Association</i>, <i>99</i>(468), 909-917. doi:10.1198/016214504000000683</li>
<li>Yang, Y. H., Dudoit, S., Luu, P., Lin, D. M., Peng, V., Ngai, J., &amp; Speed, T. P. (2002). Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation. <i>Nucleic Acids Research</i>, <i>30</i>(4), e15.</li>
<li>Yeung, K. Y., Haynor, D. R., &amp; Ruzzo, W. L. (2001). Validating clustering for gene expression data <i>Bioinformatics</i>, <i>17</i>(4), 309-318.</li>
</ol>